alcf_thetagpu/install_datascience_conda.sh

```diff
@@ -247,20 +247,21 @@ set -e
 echo "Conda install some dependencies"
 
 # note, numba pulls in numpy here too
-conda install -y cmake zip unzip astunparse ninja setuptools future six requests dataclasses graphviz numba numpy pymongo conda-build pip
-conda install -y mkl mkl-include
+conda install -c conda-forge mamba
+mamba install -y cmake zip unzip astunparse ninja setuptools future six requests dataclasses graphviz numba numpy pymongo conda-build pip
+mamba install -y mkl mkl-include
 # conda install -y cffi typing_extensions pyyaml
 
 # KGF: note, ordering of the above "defaults" channel install relative to "conda install -y -c conda-forge mamba; conda install -y pip"
 # (used to leave the pip re-install on a separate line) may affect what version of numpy you end up with
 # E.g. Jan 2023, Polaris ordering (defaults, then mamba then pip) got numpy 1.23.5 and ThetaGPU (mamba, pip, then defaults) got numpy 1.21.5
 
 # CUDA only: Add LAPACK support for the GPU if needed
-conda install -y -c pytorch magma-cuda${CUDA_VERSION_MAJOR}${CUDA_VERSION_MINOR}
+mamba install -y -c pytorch magma-cuda${CUDA_VERSION_MAJOR}${CUDA_VERSION_MINOR}
 # No magma-cuda114: https://anaconda.org/pytorch/repo
 #conda install -y -c pytorch magma-cuda116 #magma-cuda113
 
-conda install -y -c conda-forge mamba
+# conda install -y -c conda-forge mamba
```

Comment:
I moved the `mamba` install to the beginning and install the packages on 251 with `mamba` instead of `conda` (should give some speedup if no issue?)